testing/py-xxhash/APKBUILD

```diff
@@ -0,0 +1,56 @@
+# Contributor: Stuart Cardall <developer@it-offshore.co.uk>
+# Maintainer: Stuart Cardall <developer@it-offshore.co.uk>
+pkgname=py-xxhash
+_pkgname=python-xxhash
+pkgver=0.6.1
+_xxpkgname=xxHash
+_xxhashver=0.6.2
+pkgrel=0
+pkgdesc="Python Binding for xxHash"
+url="https://pypi.python.org/pypi/xxhash"
+arch="all"
+license="BSD"
+depends="python xxhash"
+makedepends="python2-dev python3-dev py-setuptools xxhash-dev"
+subpackages="py2-${pkgname#py-}:_py2 py3-${pkgname#py-}:_py3"
+source="$pkgname-$pkgver.tar.gz::https://github.com/ifduyue/$_pkgname/archive/v$pkgver.tar.gz
+	$_xxpkgname-$_xxhashver.tar.gz::https://github.com/Cyan4973/$_xxpkgname/archive/v$_xxhashver.tar.gz
```

Comment:
Is the main package mandatory dependency? Isn't `xxhash-dev` enough? If so, would it make sense to build everything from one APKBUILD?